examples/gradle-kt-dsl/lib/build.gradle.kts

```diff
@@ -0,0 +1,33 @@
+val accpVersion = "1.6.1"
```

Comment:
should we bump this to 2.0?

is there some way to define this centrall?

Reply:
This version is the one published to Maven, which is not necessarily in sync with the `version`  value in `build.gradle` for develop branch. The commit that we decide to release to Maven should bump up this version.

Reply:
i see, that makes sense.

how will we "remember" to come back and bump this? perhaps an MCM validation step to make sure that this version matches the one defined in top-level build.gradle?

Reply:
That's one of the pre-flight steps to check that the version values are updated.